docs/learn/calls.md

```diff
@@ -174,8 +183,36 @@ response = recommend_book("fantasy")
 print(response.content)
 ```
 
+In this example, we're creating a custom OpenAI client with a specific API key, organization ID, and base URL. This custom client is then passed to the call decorator, allowing us to use a custom endpoint or configuration for our LLM calls.
+
 Any custom client is supported so long as it has the same API as the original base client.
 
+## Using the `run` Method
```

Comment:
this is stale, there is no longer a `run` method since we're deprecating `BasePrompt`